### PR TITLE
Adjust MTU in `os-autoinst-setup-multi-machine`

### PR DESCRIPTION
**Pull request by Martchus**
* See added comment in diff itself for further explanation
* See https://progress.opensuse.org/issues/151612#note-23 and subsequent notes for further context

### CODE REVIEW

**Review by mimi1vx — APPROVED** [review]

**Review by okurz — APPROVED** [review]

File: script/os-autoinst-setup-multi-machine [hunk 1]
```diff
@@ -68,9 +68,11 @@ setup_multi_machine_with_networkmanager() {
 setup_multi_machine_with_wicked() {
     ovs-vsctl list-br | grep -q "$bridge" && ovs-vsctl del-br "$bridge"
     ovs-vsctl add-br "$bridge"
-    # see https://progress.opensuse.org/issues/151310
-    # The specific value comes from https://docs.openvswitch.org/en/latest/faq/issues/
-    ovs-vsctl set int $bridge mtu_request=1450
+    # Set the MUT of the bridge interface according to https://docs.openvswitch.org/en/latest/faq/issues
```
Comment (perlpunk) [inline script/os-autoinst-setup-multi-machine:71]: ?
```suggestion
    # Set the MTU of the bridge interface according to https://docs.openvswitch.org/en/latest/faq/issues
```

**Review by perlpunk — DISMISSED** [review]
.

**Comment by Martchus** [comment]
@baierjan That makes sense but I think regardless of whether NetworkManager or wicked is used we need to use `ovs-vsctl set …` here. So maybe I can unify the code paths.

**Comment by Martchus** [comment]
The tests fail with "root is not in the sudoers file." which is definitely unrelated. I'll have a look.

**Review by baierjan — APPROVED** [review]

**Pull request merged**